diff --git a/node_modules/unblocker/lib/client/unblocker-client.js b/node_modules/unblocker/lib/client/unblocker-client.js
index f66d813..d7c21de 100644
--- a/node_modules/unblocker/lib/client/unblocker-client.js
+++ b/node_modules/unblocker/lib/client/unblocker-client.js
@@ -19,6 +19,8 @@
 
   function fixUrl(urlStr, config, location) {
+    urlStr = String(urlStr);
+
     var currentRemoteHref;
     if (location.pathname.substr(0, config.prefix.length) === config.prefix) {
       currentRemoteHref =
@@ -185,13 +187,13 @@
         return new _WebSocket(
           wsProto +
-            proxyHost +
-            prefix +
-            "http" +
-            wsSecure +
-            "://" +
-            wsHost +
-            wsPath
+          proxyHost +
+          prefix +
+          "http" +
+          wsSecure +
+          "://" +
+          wsHost +
+          wsPath
         );
       }